metrics-clojure-ring/src/metrics/ring/instrument.clj

```diff
@@ -2,62 +2,107 @@
   (:require [metrics.core :refer [default-registry]]
             [metrics.counters :refer (counter inc! dec!)]
             [metrics.meters :refer (meter mark!)]
-            [metrics.timers :refer (timer time!)])
+            [metrics.timers :refer (timer time!)]
+            [clojure.string :as string])
   (:import [com.codahale.metrics MetricRegistry]))
 
+(def default-options
+  {:prefix []})
 
 (defn- mark-in! [metric-map k]
   (when-let [metric (metric-map k (metric-map :other))]
     (mark! metric)))
 
+(defn- ring-metrics [reg {:keys [prefix] :as options}]
```

Comment:
Sorry to nitpick but can you please move the argument vector to the next line? We don't do this consistently in this project
but every function that has a doc string or has multiple heads (overrides) does that.